src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -275,8 +274,10 @@ uct_rdmacm_cm_reserved_qpn_blk_alloc(uct_rdmacm_cm_t *cm,
     char out[UCT_IB_MLX5DV_ST_SZ_BYTES(general_obj_out_cmd_hdr)] = {};
     struct uct_rdmacm_cm_reserved_qpn_blk *blk;
     void *attr;
+    uint32_t qpns_per_obj;
 
-    if(ctx->reserved_qpn_ctx.reserved_qpn_granularity > (sizeof(blk->bmp) << 3)) {
+    qpns_per_obj = 1 << ctx->reserved_qpn_ctx.log_reserved_qpn_granularity;
+    if(qpns_per_obj > (sizeof(blk->bmp) << 3)) {
```

Comment:
pls add a function or macro for `1 << ctx->reserved_qpn_ctx.log_reserved_qpn_granularity` and `(sizeof(blk->bmp) << 3)`

Reply:
@evgeny-leksikov Done.